src/libraries/icubmod/cartesianController/CMakeLists.txt

```diff
@@ -37,7 +41,11 @@ if(NOT SKIP_cartesiancontrollerclient)
    target_link_libraries(cartesiancontrollerclient iKin ${YARP_LIBRARIES})
 
    icub_export_plugin(cartesiancontrollerclient)
-   yarp_install(FILES cartesiancontrollerclient.ini DESTINATION ${ICUB_PLUGIN_MANIFESTS_INSTALL_DIR})
+     yarp_install(TARGETS cartesiancontrollerclient
+               COMPONENT Runtime
+               LIBRARY DESTINATION ${ICUB_DYNAMIC_PLUGINS_INSTALL_DIR}
+               ARCHIVE DESTINATION ${ICUB_STATIC_PLUGINS_INSTALL_DIR}
+               YARP_INI DESTINATION ${ICUB_PLUGIN_MANIFESTS_INSTALL_DIR})
```

Comment:
Please double check if the generation works properly when there are 2 .ini files generated in the same CMakeLists.txt.

Reply:
yes, two separated files are are generated correctly